### ISSUE TITLE
Catalog remains incomplete if advanced before scan complete during setup

### ISSUE THREAD

**Issue by ryanfeeley** [quicksilver/Quicksilver#2525]
**Describe the bug**
Application catalog remains incomplete if setup is advanced before the scan is complete. Applications like FaceTime and Calculator are not indexed. Took me a while to figure this one out.

**To Reproduce**
1. Run Setup
2. Before scan is complete, while the spinning wheel is beside Catalog, click Continue

**Expected behavior**
Continue button is not enabled until scan is complete OR scan is completed asynchronously.

**Screenshots**
<img width="442" alt="quicksilver" src="https://user-images.githubusercontent.com/68213/86245744-ebcffc00-bb77-11ea-898e-79adc7c65861.png">

**What version of macOS?**
Catalina

**What version of Quicksilver?**
1.6.0

**Comment by skurfer** [comment]
Good catch. What happens if you call up Quicksilver and force a rescan of the entire catalog with ⌘R?
